app/src/main/kotlin/app/youkai/ui/feature/settings/view/PreferenceView.kt

```diff
@@ -0,0 +1,138 @@
+package app.youkai.ui.feature.settings.view
```

Comment:
We always had custom views under `widget` and `view` was more for smaller view-related classes. Maybe we should stick to that?